### PR TITLE
Bug Fix Speech2Srt tutorial

### PR DESCRIPTION
**Pull request by anantdamle**
used a fixed version for Google client libraries as newer version may have different API
Fix timeout exception in translate_txt.py to wait for longer duration

### CODE REVIEW

**Comment by anantdamle** [comment]
@lepistom can you please review the proposed changes to your tutorial
@ToddKopriva this is not a new tutorial just code fixes for one of the tutorials

**Review by ToddKopriva — APPROVED** [review]
This looks good to me, but I'll defer to @lepistom for the final approval.

**Comment by Rohit484** [comment]
Google client libraries

File: tutorials/speech2srt/requirements.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
-google-cloud-speech>=1.2.0
-google-cloud-translate>=2.0.0
-srt>=3.0.0
+google-cloud-speech==1.2.0
+google-cloud-translate==3.0.0
```
Comment (anantdamle) [inline tutorials/speech2srt/requirements.txt:2]: @lepistom Using a fixed version ensures the changes in API surface doesn't break the existing code.

File: tutorials/speech2srt/speech2srt.py [hunk 2]
```diff
@@ -68,10 +63,7 @@ def break_sentences(args, subs, alternative):
     for w in alternative.words:
         if firstword:
             # first word in sentence, record start time
-            start_hhmmss = time.strftime('%H:%M:%S', time.gmtime(
-                w.start_time.seconds))
-            start_ms = int(w.start_time.microseconds / 1000)
-            start = start_hhmmss + "," + str(start_ms)
```
Comment (anantdamle) [inline tutorials/speech2srt/speech2srt.py:74]: @lepistom the SRT library and the speech proto  both support `timedelta` there is no need to convert to string and parse back.

File: tutorials/speech2srt/translate_txt.py [hunk 3]
```diff
@@ -44,33 +43,49 @@ def batch_translate_text(
     input_configs = [input_configs_element]
     gcs_destination = {"output_uri_prefix": output_uri}
     output_config = {"gcs_destination": gcs_destination}
-    parent = client.location_path(project_id, location)
```
Comment (anantdamle) [inline tutorials/speech2srt/translate_txt.py:47]: @lepistom changes in the API surface requires this change.

File: tutorials/speech2srt/speech2srt.py [hunk 4]
```diff
@@ -139,13 +127,23 @@ def main():
...
+    parser.add_argument(
+        "--encoding",
+        type=str,
+        default='LINEAR16'
+    )
+    parser.add_argument(
+        "--audio_channel_count",
+        type=int,
+        default=1
+    )
```
Comment (anantdamle) [inline tutorials/speech2srt/speech2srt.py:146]: @lepistom Added these parameters to support other file types and multi-channel audio files.

**Comment by Rohit484** [comment]
API

**Comment by anantdamle** [comment]
> Google client libraries

> API

@Rohit484 Can you clarify on what do you mean by the two comments? Is there something I could add something to this PR?

**Review by lepistom — APPROVED** [review]
ack

**Comment by lepistom** [comment]
merging updates

**Comment by ToddKopriva** [comment]
Thanks, @lepistom .

**Pull request merged**